suite-native/module-settings/src/screens/SettingsViewOnly.tsx

```diff
@@ -1,33 +1,117 @@
+import { useState } from 'react';
+
+import { A } from '@mobily/ts-belt';
 
 import { Screen, ScreenSubHeader } from '@suite-native/navigation';
 import { Box, Button, Card, HStack, IconButton, Text } from '@suite-native/atoms';
 import { Translation, useTranslate } from '@suite-native/intl';
 import { selectPhysicalDevices, toggleRememberDevice } from '@suite-common/wallet-core';
 import { TrezorDevice } from '@suite-common/suite-types';
+import { Link } from '@suite-native/link';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { useAlert } from '@suite-native/alerts';
+import { useToast } from '@suite-native/toasts';
 
-export const SettingsViewOnly = () => {
+import { AboutViewOnlyBottomSheet } from '../components/AboutViewOnlyBottomSheet';
+import { SecuredCoinsSvg } from '../assets/SecuredCoinsSvg';
+
+type AboutProps = {
+    onPressAbout: () => void;
+};
+
+const About = ({ onPressAbout }: AboutProps) => (
+    <Box marginVertical="medium" marginHorizontal="extraLarge" alignItems="center">
+        <Text variant="hint" textAlign="center" color="textSubdued">
+            <Translation
+                id="moduleSettings.viewOnly.subtitle"
+                values={{
+                    about: chunk =>
+                        chunk && (
+                            <Link
+                                label={chunk}
+                                onPress={onPressAbout}
+                                isUnderlined
+                                textColor="textSubdued"
+                                textPressedColor="textSubdued"
+                            />
+                        ),
+                }}
+            />
+        </Text>
+    </Box>
+);
+
+const svgWrapperStyle = prepareNativeStyle(utils => ({
+    width: 180,
+    height: 156,
+    paddingBottom: utils.spacings.large,
+}));
+
+const emptyContentStyle = prepareNativeStyle(utils => ({
+    flex: 1,
+    alignItems: 'center',
+    justifyContent: 'center',
+    paddingHorizontal: utils.spacings.large,
+}));
+
+const EmptyContent = ({ onPressAbout }: AboutProps) => {
+    const { applyStyle } = useNativeStyles();
+
+    return (
+        <Box style={applyStyle(emptyContentStyle)}>
+            <Box style={applyStyle(svgWrapperStyle)}>
+                <SecuredCoinsSvg />
+            </Box>
+            <Text variant="titleSmall" textAlign="center">
+                <Translation id="moduleSettings.viewOnly.emptyTitle" />
+            </Text>
+            <About onPressAbout={onPressAbout} />
+        </Box>
+    );
+};
+
+const DevicesContent = ({ onPressAbout }: AboutProps) => {
     const dispatch = useDispatch();
     const { translate } = useTranslate();
-
     const devices = useSelector(selectPhysicalDevices);
+    const { showAlert, hideAlert } = useAlert();
+    const { showToast } = useToast();
 
-    const handleViewOnlyChange = (device: TrezorDevice) => {
+    const toggleViewOnly = (device: TrezorDevice) => {
```

Comment:
```suggestion
    const toggleViewOnly = (isDeviceRemembered: boolean) => {
```

The rest of device object is not needed here.

Reply:
dispatching `toggleRememberDevice` requires device object